stats/view/worker.go

```diff
@@ -233,6 +233,7 @@ func (w *worker) reportView(v *viewInternal, now time.Time) {
 		e.ExportView(viewData)
 	}
 	exportersMu.Unlock()
+	v.resetValues()
```

Comment:
1. This should be also called from w.Read() method. This is for new mechanism (ExportMetrics)
2. Apart from that reportUsage() should not go through all the views unless there is an exporter registered (used for old mechanism ExportView())
If user configures both ways or multiple exporter using new mechanism then values of min/max is unpredictable.
3. Min/Max doc should be updated to reflect the behavior.
4. Add tests for min/max.

Reply:
@rghetia done, let me know if the last commit is what you were looking for :+1:

Reply:
test is good.
Item 2 above is not taken care of.

There should be new option in ReadAndExport method. If the option to reset min/max is specified then only do the reset.

Similarly, for old approach (export viewdata) there should be a new method view.ResetMinMaxOnExport() - It is rather a long name but I don't have a good suggestion. Alternatively, provide reset functionality only using new approach with ReadAndExport().